src/barril/units/_tests/test_scalar.py

```diff
@@ -1,15 +1,11 @@
 """
-    REMARKS: Unicode
-        The "unit" must not containt unicode character. All unit name and representation must use
-        the standard ascii representation.
+    The 'unit' must not containt str character. All unit name and representation must use
```

Comment:
```suggestion
    'unit' must be ascii only.
```

And delete the line below

Reply:
Done